youtube-search-with-react/src/App.js

```diff
@@ -1,27 +1,47 @@
 import React, { Component } from 'react';
+
 import YTSearch from 'youtube-api-search';
+
 import './App.css';
 import { myConfig } from './config';
+import SearchBar from './components/SearchBar.js';
+import VideoDetail from './components/VideoDetail.js';
+import VideoList from './components/VideoList.js';
+
 
 const key = myConfig.API_KEY;
 
 class App extends Component {
-  render() {
-   YTSearch({ key: key, term: 'hello' }, (data) => {
-        console.log(data)
-   });
-
-    return (
-      <div className="App">
-        <div className="App-header">
-          <h2>Welcome to React</h2>
-        </div>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            videos: [],
+            selectedVideo: null,
+            };
+        this.videoSearch('hello');
+    }
+
+    videoSearch(term) {
+        YTSearch({key: key, term: term}, (videos) => {
+            this.setState({
+                videos: videos,
+                selectedVideo: videos[0],
```

Comment:
I think the default behavior should be to just display a list of videos and upon selection of a video by the use we should show video detail component.